Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -494,6 +493,11 @@ void WebKitWebViewClient::didReceiveUserMessage(WKWPE::View&, UserMessage&& mess
 {
     webkitWebViewDidReceiveUserMessage(m_webView, WTFMove(message), WTFMove(completionHandler));
 }
+
+WebKitWebResourceLoadManager* WebKitWebViewClient::webResourceLoadManager()
```

Comment:
It should return `WebKitWebResourceLoadManager&` IMO, because once the object is constructed it should never be `nullptr`. This will make it easier to keep track of what is and isn't nullable. You can add an assert to check that it's not null before dereferencing here.

(I think it's worth doing this for GObject types too, since it's a nice safety improvement. But we've intentionally avoided that so far. This one is not a GObject, though. ;)

Reply:
No, it's nullptr when GLib API is not used (WTR)